pkg/data/management/role_data.go

```diff
@@ -46,7 +46,8 @@ func addRoles(wrangler *wrangler.Context, management *config.ManagementContext)
 	rb.addRole("Manage Cluster Drivers", "kontainerdrivers-manage").
 		addRule().apiGroups("management.cattle.io").resources("kontainerdrivers").verbs("*")
 	rb.addRole("Manage Catalogs", "catalogs-manage").
-		addRule().apiGroups("management.cattle.io").resources("catalogs", "templates", "templateversions").verbs("*")
+		addRule().apiGroups("management.cattle.io").resources("catalogs", "templates", "templateversions").verbs("*").
+		addRule().apiGroups("catalog.cattle.io").resources("clusterrepos").verbs("*")
```

Comment:
I'm confused about something. This role grants access to global catalogs, and not clustercatalogs or projectcatalogs.  In the new system there are only clusterrepos, which are the equivalent of both catalogs and clustercatalogs. 
```
Create a standard user, check the "Configure Catalogs" builtin role
Add new user as a member to a cluster (local or downstream)
Log in as user and go to Apps & Marketplace > Chart Repositories in the cluster
```
Doesn't `clustercatalogs-manage` actually make more sense ? Or I could see both.